demo/services/availableComponentBuilder.ts

```diff
@@ -0,0 +1,32 @@
+import {AvailableComponent, default as AvailableComponentsService} from './availableComponentsService';
```

Comment:
You probably want to rename this accordingly to the new name :wink:

Reply:
That's ok, it's used in `componentFactory` and in decorator function.